rudder-agent/SOURCES/Makefile

```diff
@@ -607,6 +610,10 @@ build-cfengine-stamp:
 	cd cfengine-source && ./configure --prefix=$(RUDDER_DIR) --with-workdir=/var/rudder/cfengine-community --enable-static=yes --enable-shared=no --without-postgresql --without-mysql $(ACL_ARG) $(LMDB_ARG) $(OPENSSL_ARG) $(PCRE_ARG) $(CURL_ARG) $(YAML_ARG) $(XML_ARG) $(PAM_ARG) CFLAGS="$(CFLAGS) $(PIE_CFLAGS)" LDFLAGS="$(LDFLAGS)" CPPFLAGS="-I$(CURDIR)/dependencies$(RUDDER_DIR)/include/libxml2"
 	cd cfengine-source && $(MAKE) $(MAKE_J)
 	cd cfengine-source && $(MAKE) $(MAKE_J) install DESTDIR=$(CURDIR)/dependencies
+# solaris doesn't have uuidgen and needs one
+ifeq (solaris,$(OS_FAMILY))
+	gcc -luuid rudder-uuidgen.c -o rudder-uuidgen
```

Comment:
don't we already have different uuid generation for AIX?

Reply:
no, aix has an uuidgen binary in the system that can be called from the rudder-uuidgen script

Reply:
so we need to install into the destdir then?

Reply:
in fact it has the same name as the script so it replaces it, so no need to change the install part